packages/core/src/node/runtimeModule/runtimeConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { RuntimeModuleID, type VirtualModulePlugin } from './types';
+
+/**
+ * some configuration in `rspress.config.ts` serialized from compile time side to runtime
+ */
+export const runtimeConfigVMPlugin: VirtualModulePlugin = context => {
+  return {
+    [RuntimeModuleID.RuntimeConfig]: () => {
+      const { config } = context;
+
+      const { base } = config;
+
+      return `
+export const base = ${JSON.stringify(base)};`;
```

Comment:
```suggestion
      return `export const base = ${JSON.stringify(base)};`;
```

It's weird to have starting blanks without ending blanks?